content/spin/api-guides-overview.md

```diff
@@ -0,0 +1,24 @@
+title = "API Guides Overview"
+template = "spin_main"
+date = "2023-03-03T03:03:03Z"
+
+---
+
+### Spin's Hosting Capabilities and Interfaces
+
+Spin is a multi-language framework for [writing server-like applications](https://developer.fermyon.com/spin/writing-apps) that compile into WebAssembly (Wasm). The Spin framework provides capabilities for your Wasm-powered application by offering a set of interfaces. For example, making sufficient HTTP functionality available to [send outbound HTTP requests](https://developer.fermyon.com/spin/http-outbound) and more.
+
+The following table shows the status of capabilities/interfaces that you can use to build and run real-world applications with Wasm.
+
+| Host Capabilities/Interfaces           | Stability  |    Cloud    |
+|----------------------------------------|----------|-------|
+| [HTTP](https://developer.fermyon.com/spin/http-trigger)                          | Stable   | Yes   |
+| [Redis](https://developer.fermyon.com/spin/redis-trigger)                         | Stable   | No  |
+| [Outbound HTTP](https://developer.fermyon.com/spin/rust-components.md#sending-outbound-http-requests)                          | Stable   | Yes   |
+| [Outbound Redis](https://developer.fermyon.com/spin/rust-components.md#redis-components)                         | Stable  | Yes   |
+| [Dynamic config](https://developer.fermyon.com/spin/dynamic-configuration)                         | Experimental | Coming soon |
+| [PostgreSQL](https://developer.fermyon.com/spin/rdbms-storage)                             | Experimental | Yes |
```

Comment:
Should we call these Outbound PostgreSQL and Outbound MySQL? I think that might be more expressive. Wdyt?

Reply:
Hi @michelleN 
Great question, in the context of the "language support overview" we decided to drop the word "outbound" so happy to do the same here.
![Screenshot 2023-03-21 at 1 56 13 pm](https://user-images.githubusercontent.com/9831342/226514139-f453b1b1-dca3-4bbb-a152-ab8f9e423f4f.png)